File: AvitoTest/app/src/main/java/avito/test/data/model/GitData.java
package avito.test.data.model;

import java.util.ArrayList;
import java.util.List;

public class GitData {

  private List<User> userList;
  private List<Repo> repoList;

  public List<User> getUserList() {
    if (userList == null)
      userList = new ArrayList<>();
    return userList;
  }

  public void setUserList(List<User> userList) {
    this.userList = userList;
  }

  public List<Repo> getRepoList() {
    if (repoList == null)
      repoList = new ArrayList<>();
    return repoList;
  }

  public void setRepoList(List<Repo> repoList) {
    this.repoList = repoList;
  }
}

File: AvitoTest/app/src/main/java/avito/test/data/model/GitData.kt
package avito.test.data.model

import java.util.ArrayList

public class GitData {

    private var userList: List<User>? = null
    private var repoList: List<Repo>? = null

    public fun getUserList(): List<User> {
        if (userList == null) userList = ArrayList<User>()
        return userList
    }

    public fun setUserList(userList: List<User>) {
        this.userList = userList
    }

    public fun getRepoList(): List<Repo> {
        if (repoList == null) repoList = ArrayList<Repo>()
        return repoList
    }

    public fun setRepoList(repoList: List<Repo>) {
        this.repoList = repoList
    }
}
